composer.json

```diff
@@ -4,7 +4,8 @@
   "description": "PHP server library for WOVNio",
   "homepage": "https://wovn.io",
   "require": {
-    "php": ">=5.3"
+    "php": ">=5.3",
+    "ext-fileinfo": "*"
```

Comment:
This is the first time we require a dependency for WOVN.php. I don't think any of our customer use composer for WOVN.php installation, so we need to make it clear. Please talk with Swappers and OnSite to decide the best strategy to adopt. In the long run, publishing WOVN.php so it can be installed from composer would be the best.

Reply:
This "dependency" is kind of special is that it is a PHP extension module that is shippped by default for PHP 5.3+, which also coincides with the PHP versions WOVN.php targets. So, strictly speaking no installation is required and the user needs to at most enable this feature. I think this can be done by adding a line in the installation instruction.

Reply:
I see, looks good, thanks.